index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head><link rel="stylesheet" href="style.css">
+    <title>BFHS-Open | Ben Franklin Computer Science Club</title>
+    <meta charset="UTF-8">
+  </head>
+
+  <body>
+
+    <nav>
+      <a href="index.html">BFHS-Open</a>
+      <a href="about.html">About</a>
+    </nav>
+
+    <p>A center of gravity for open work happening at or associated with Benjamin Franklin High School.</p>
+    <p>BHFS-Open is an initiative of the BFHS Computer Science Club in effort to create, organize, and support projects being done transparently.</p>
+
+    <h1>Get involved</h1>
+    <ul>
+      <li>Attend weekly meetings in Room 213</li>
+      <ul>
+        <li>Mondays after school and Mondays and Wednesdays during lunch</li>
+      </ul>
+      <li>Join the CSC <a href="https://discord.gg/kUd3m8HMG8">Discord Server</a></li>
+      <li>(coming soon) Join our IRC channel at #bfhs-open on libera.chat</li>
+    </ul>
+
+    <h1>How to participate</h1>
+    <ul>
+      <li><a href="https://github.com/BFHS-Open">Contribute to our various projects on Github</a></li>
+      <li><a href="https://github.com/BFHS-Open/knowledge-base">Create your own knowledge base and share it with us</a></li>
+      <li><a href="https://bfhs-open.github.io/ctf/">Enter into competitions with us</a></li>
```

Comment:
Does the /ctf/ directory currently point anywhere?

Reply:
I was looking at the Competitions page, and it read as though it was an upcoming event. Do you think we should rewrite it's content, and include it in this pull request, or remove that link and save it for a future pull request?

Reply:
Yeah I would probably remove it for the time being and reintroduce it at a later date.

```suggestion
```